recipe_scrapers/kptncook.py

```diff
@@ -16,12 +15,7 @@
 class KptnCook(AbstractScraper):
     def __init__(self, url, *args, **kwargs):
         super().__init__(url, *args, **kwargs)
-        if urlparse(url).hostname == "mobile.kptncook.com":
-            parsed_url = urlparse(url)
-        else:
-            # If it's a sharing.kptncook.com/* link we first need to follow the http forwards to get the recipe ID
-            recipe_request = requests.get(url, headers=HEADERS)
-            parsed_url = urlparse(recipe_request.url)
+        parsed_url = urlparse(self.url)
```

Comment:
Neat - this seems a lot simpler logically, and solves the duplicate-request problem.  Thank you!

Resolves https://github.com/hhursev/recipe-scrapers/pull/503#discussion_r901527643